vpr/src/base/read_options.cpp

```diff
@@ -2461,6 +2461,13 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_grp.add_argument(args.custom_3d_sb_fanin_fanout, "--custom_3d_sb_fanin_fanout")
```

Comment:
This seems like it would fit better in the arch file (as a new parameter) than as a command line parameter.  What do you think?

Reply:
I also think that it might be better to put it in the architecture file, but for now I just keep it as it is to make the testing easier with command line option. When the branch is final, I will talk to you about the architecture change.